static/src/js/builder/codelistbuilder.jsx

```diff
@@ -413,12 +416,20 @@ function SearchForm(props) {
       </div>
       <div>
         <small className="form-text text-muted">
-          <p>
-            To search by code, prefix your search with <code>code:</code>. For
-            instance, use <code>code:xyz</code> to find the concept with code{" "}
-            <code>xyz</code>, or <code>code:xyz*</code> to find all concepts
-            with codes beginning <code>xyz</code>.
-          </p>
+          {codingSystemName === "ICD-10" ? (
+            <p>
+              To search by code, prefix your search with <code>code:</code>. For
+              instance, use <code>code:xyz</code> to find the concept with code{" "}
+              <code>xyz</code>, or <code>code:xyz*</code> to find all concepts
+              with codes beginning <code>xyz</code>.
```

Comment:
Could you add something like "(Prefix search only available for ICD-10.)"?

Reply:
Just to the ICD-10 version?  Or to all?
"prefix search" might be confusing, since the text already refers to prefixing searches with "code:" to search by code.  "wildcard search" instead?

Reply:
> Just to the ICD-10 version? Or to all?

I think to just the ICD-10 version.

And "wildcard search" is better, thanks.